jujugui/static/gui/src/app/app.js

```diff
@@ -1361,6 +1361,7 @@ YUI.add('juju-gui', function(Y) {
           getUser={this.payment && this.payment.getUser.bind(this.payment)}
           getUserName={getUserName}
           gisf={this.get('gisf')}
+          githubSSHKeys={window.jujugui.sshKeys.githubSSHKeys}
```

Comment:
plz apply this change to the init code too...this change will need to be done in the component-renderers-mixin.js